src/components/welcome-page/Welcome.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../styles/variables.scss";
```

Comment:
in the whole file you should take care off indendations, enters etc because it looks really bad

Reply:
Refers to whole .scss file -> you shouldn't target by tags

Reply:
Another global comment: every font-size, font-weight and colors should be stored in variables.scss or other reasonable file (like FontWeights.scss or something like that)